### DISCUSSION TITLE
Add a way to copy a plugin's composer package name to control panel

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#16281] [Ideas] [answered]
Many developers such as myself prefer to run plugin updates using the composer CLI instead of in the control panel, especially when performing larger updates. I'd love to see a way to copy the package name (e.g. `craftcms/feed-me`) to the clipboard on both the "available updates" and "installed plugins" pages to make it less of a hassle.

**Comment by brandonkelly** [comment, marked as answer]
Good suggestion! Just added copy actions to the Plugins index page and Updates utility, for Craft 5.6. (8bdff8362c1631cb4de400b3feace218e31e5a38)

<p><img src="https://github.com/user-attachments/assets/770ee2a6-a61b-47b3-a527-ed93c2dd8214" width="206" height="135" alt="A plugin’s action menu within the Updates utility"></p>

<p><img src="https://github.com/user-attachments/assets/d19c94a0-df8d-4971-8d95-03492a392025" width="220" height="243" alt="A plugin’s action menu within the Plugins index page"></p>

**Reply by brandonkelly** [reply]
Craft 5.6.0 is out now 🚀

**Reply by Antimated** [reply]
Is it possible that this update removed the "Update all" button? I just installed Craft 5.6.0.2 (from 5.5.10) and suddenly the button is gone.

![Screenshot 2025-01-22 at 16 02 47](https://github.com/user-attachments/assets/d998b3af-3ed2-45cc-97e7-f7a56799a98c)

**Reply by brandonkelly** [reply]
@Antimated Thanks for pointing that out. Fixed now in 5.6.2.
